DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,11 +164,15 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
```

Comment:
@aminnj Nick, please follow up (check in 76X and make a fix PR for 76X; and then update the 74X/75X PRs)